src/bootstrap-table.js

```diff
@@ -757,7 +757,7 @@ class BootstrapTable {
       this.$toolbar.append(html.join(''))
       const $searchInput = this.$toolbar.find('.search input')
       const handleInputEvent = () => {
-        const eventTriggers = `keyup drop blur ${Utils.isIEBrowser() ? 'mouseup' : ''}`
+        const eventTriggers = 'keyup drop blur mouseup'
```

Comment:
No need to judge whether it is IE?

Reply:
No, since using search as input type we need to listen the mouseup because that event is fired when the x icon is clicked